config/boot.rb

```diff
@@ -5,7 +5,7 @@
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
 begin
-  require 'bootsnap/setup' if ENV['RAILS_ENV'] != 'production' && ENV['ENABLE_BOOTSNAP'] != 'false'
+  require 'bootsnap/setup' if ENV['ENABLE_BOOTSNAP'] != 'false'
```

Comment:
this is too early for us to use `IdentityStore.config` right? Where do we control this env var?

Reply:
Yep 😕

Right now, it doesn't get set anywhere. It's possible to put some stuff into the build script to allow environments to enable/disable it, but my hope is it mostly just works or is at least able to fail gracefully.